### PR TITLE
Code cleanup and fixes

### PR DESCRIPTION
**Pull request by GingerAvalanche**
Most notable fix is to aura_mask_0, which should now be set on all characters with mage_perk

Secondary notable fix is to aura spells, which should now properly state that they decrease mana gen when starting them, instead of claiming to increase it. The effect always happened correctly, but the tooltip was wrong.

### CODE REVIEW

File: gui/magic_texticons.gui [hunk 1]
```diff
@@ -140,13 +140,3 @@ texticon = {
 		fontsize = 16
 	}
 }
-
-texticon = {
```
Comment (Donekulda) [inline gui/magic_texticons.gui:144]: so you moved from mana cost to just inactive aura ?

File: localization/english/ancient_magic_game_concepts_l_english.yml [hunk 2]
```diff
@@ -1,102 +1,76 @@
...
 
  game_concept_mana: "Mana"
- game_concept_mana_i:0 "@aura_icon_active!"
+ game_concept_mana_i: "@aura_icon_active!"
  game_concept_mana_desc: "Mana is a fundamental force in the universe. It is a raw expression of 'potential' which may be used by a skilled magical practitioner to bend the laws of reality to their will.\n\nWhenever a mage casts a spell, they are channeling this 'potential' into an outcome of their own choosing, expending their mana reserves in the process. The more powerful an effect a [GetTrait('mage').GetName( GetNullCharacter ) |LVl] wishes to bring about, the more mana that effect requires."
  game_concept_magical_power: "Magic Power"
 
  game_concept_mana_cost: "Mana Cost"
- game_concept_mana_cost_i: "@icon_mana_cost!"
+ game_concept_mana_cost_i: "@aura_icon_inactive!"
```
Comment (Donekulda) [inline localization/english/ancient_magic_game_concepts_l_english.yml:28]: where are the icons comming from ? I dont see anywhere their declaration

Comment (GingerAvalanche) [reply localization/english/ancient_magic_game_concepts_l_english.yml:28]: Top of `gui/magic_texticons.gui`

Comment (Donekulda) [reply localization/english/ancient_magic_game_concepts_l_english.yml:28]: where i dont see it there?

Comment (Donekulda) [reply localization/english/ancient_magic_game_concepts_l_english.yml:28]: i mean i see this there but i dont see there some of the icons

File: gui/magic_texticons.gui (hunk 1 above)
Comment (GingerAvalanche) [reply gui/magic_texticons.gui:144]: It was actually always there, under that name.

**Pull request merged**